src/Raven.Server/Commercial/LicenseValidator.cs

```diff
@@ -14,7 +14,9 @@ public static class LicenseValidator
         private static readonly string[] Terms =
         {
             "type", "version", "expiration", "updatesExpiration",
-            "RAM", "cores", "ISV", "encryption", "fips", "monitoring"
+            "RAM", "cores", "ISV", "encryption", "fips", "monitoring",
```

Comment:
fips can be removed.
ISV change to "redist"
RAM change to memory

Reply:
done